### PR TITLE
added the map, fixes #1

### LINKED ISSUE #1: Pokemon Map
Must include all the regions

### PR DESCRIPTION
**Pull request by Thunderducky**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -149,5 +149,10 @@ <h1>Best Starters in Pokemon</h1>
         </div>
       </div>
     </div>
+    <script>
+      setInterval(() => {
+        alert("Grookey for life")
```
Comment (baggiodev) [inline index.html]: This is bad and you should feel bad.

**Review by baggiodev — CHANGES_REQUESTED** [review]
No alerts thanks

File: index.html (hunk 1 above)
Comment (Thunderducky) [reply index.html]: that's fair

**Comment by Thunderducky** [comment]
I've updated it at your request

**Review by baggiodev — APPROVED** [review]
Good job dig the new bootstrap styles!

**Pull request merged**